lib/pull.js

```diff
@@ -22,16 +29,34 @@ async function downloadAndSave(projectConfig, token) {
     });
     fs.writeFileSync(consts.TEXT_FILE, JSON.stringify(res.data, null, 2));
     msg = `Fetching the latest text from ${output.info(projectName)}... ${output.success('done')}. \nSuccessfully saved to ${output.info(consts.TEXT_FILE)}.`;
+    spinner.stop();
+    return console.log(msg);
   } catch (e) {
+    spinner.stop();
     let error = e.message;
+    if (e.response && e.response.status === 404) {
+      await askForAnotherToken();
+      pull();
+      return;
+    }
     if (e.response && e.response.status === 401) {
-      error = output.info("Looks like you don't have access to the specified project.");
+      error = "You don't have access to the selected project";
+      msg = `${output.errorText(error)}.\nChoose another using the ${output.info('project')} command, or update your API key.`;
+      return console.log(msg);
     }
-    msg = output.errorText(`We hit an error fetching text from the project: ${error}`);
+    if (e.response && e.response.status === 403) {
+      error = "The requested project doesn't have Developer Mode enabled";
+      msg = `${output.errorText(error)}.\nPlease choose a different project using the ${output.info('project')} command, or turn on Developer Mode for this project. Learn more here: ${output.subtle('https://www.dittowords.com/docs/ditto-developer-mode')}.`;
+      return console.log(msg);
+    }
+    if (e.response && e.response.status === 400) {
+      error = "project not found";
+      msg = `We hit an error fetching text from the project: ${output.errorText(error)}.\nChoose another using the ${output.info('project')} command.`;
```

Comment:
nit - we can take out / combine this and the next line as it's the same as the ones after the if {}